projects/distributed-tracing/src/shared/dashboard/dashboard-wrapper/application-aware-dashboard.component.ts

```diff
@@ -36,6 +35,9 @@ export class ApplicationAwareDashboardComponent implements OnDestroy {
   @Input()
   public editable: boolean = false;
 
+  @Input()
+  public padding?: number;
```

Comment:
Feels real hacky. Can we not pad this through external styles?

Reply:
Agree it's better than overriding the class with ng-deep. I guess this is the tradeoff - bring your own classes and get inconsistency, or build them in and need hacky overrides.